.github/workflows/push-main.yml

```diff
@@ -76,6 +76,12 @@ jobs:
           # Any step using skopeo needs SKOPEO_SYNC_FILES env var
           SKOPEO_SYNC_FILES=$(mktemp -d)
           echo "SKOPEO_SYNC_FILES=${SKOPEO_SYNC_FILES}" >> $GITHUB_ENV
+
+          # limit what we push as time stamped images to images built in the current year and month only
+          current_year=$(date +%Y)
+          current_month=$(date +%m)
```

Comment:
TIL env vars are case-insensitive

Reply:
I can rename and make them upper case, if you like? I opted to leave as lower because we're just using them "here".